charts/karpenter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: karpenter
 description: A Helm chart for Karpenter, an open-source node provisioning project built for Kubernetes.
 type: application
-version: 0.21.1
+version: 0.21.2
```

Comment:
Can you revert these changes? It's circumventing our normal release process.

Reply:
Done now, sorry wasn't completely clear on the release process